align_system/algorithms/outlines_adm.py

```diff
@@ -121,6 +131,36 @@ def kdma_value_to_system_prompt(kdma, value):
                 return low_maximization_system_prompt()
             else:
                 return high_maximization_system_prompt()
+        elif kdma == "ProtocolFocus":
+            if value == 0.0:
+                return low_protocol_focus_system_prompt()
+            else:
+                return high_protocol_focus_system_prompt()
```

Comment:
Curious why we're only using the "low" prompt when the value is `0.0` (instead of splitting the range in half, e.g. `< 0.5` as above)?

Reply:
The only values for the NAACL24 KDMAs are 0.0 and 10.0 for high and low respectively. But the `fairness` attribute has a value of 5.0 to indicate a balanced decision. But at the moment, we are only focusing on the high and low prompt cases here.

Reply:
Looking to future proof this a little bit (I would set it to a `< 0.5` as low, and otherwise set it to high) the current set up is not intuitive and I can see it biting us in the future when we're wondering why we're getting a high prompt when we have it set to `0.1` if we ever re-run experiments around this stuff